client/src/Components/CommonComponent/Table/Table.jsx

```diff
@@ -24,9 +55,11 @@ class TableCmponent extends Component {
 
   render() {
 
-    const { viewPopup, ViewPopup, editPopup, EditPopup, deletePopup, DeletePopup, viewHtml, editHtml, deleteHtml } = this.props;
+    const { viewPopup, editPopup, EditPopup, deletePopup, DeletePopup, viewHtml, editHtml, deleteHtml } = this.props;
     const { Column } = Table;
-    const { tableData: columns} = this.state;
+    const { tableData: columns, singleCustomer: { id } } = this.state;
+    const { singleCaptain } = this.state;
```

Comment:
@naremanhilles Very minor thing: You could combine line 60 and 61:
`const { singleCaptain, tableData: columns, singleCustomer: { id } } = this.state;`